src/lib/bcmp/bcmp_time.cpp

```diff
@@ -104,12 +103,20 @@ static void bcmp_time_process_time_request_msg(const BcmpSystemTimeRequest *msg)
 */
 static void bcmp_time_process_time_set_msg(const BcmpSystemTimeSet *msg) {
   UtcDateTime datetime;
-  // TODO: make this an abstraction
   date_time_from_utc(msg->utc_time_us, &datetime);
+<<<<<<< HEAD
+  RtcTimeAndDate time = {.year = datetime.year,
+                         .month = datetime.month,
+                         .day = datetime.day,
+                         .hour = datetime.hour,
+                         .minute = datetime.min,
+                         .second = datetime.sec,
+=======
   RtcTimeAndDate time = {// TODO: Consolidate the time functions into util.h
                          .year = datetime.year,       .month = datetime.month,
                          .day = datetime.day,         .hour = datetime.hour,
                          .minute = datetime.min,      .second = datetime.sec,
+>>>>>>> experiment/bm_core
```

Comment:
Looks like some conflicts sneaking through here

Reply:
woops forgot to commit that one!